packages/calcite-components/src/components/tree-item/tree-item.e2e.ts

```diff
@@ -450,6 +450,47 @@ describe("calcite-tree-item", () => {
     expect(collapseSpy).toHaveReceivedEventTimes(1);
   });
 
+  describe("selection event", () => {
```

Comment:
Since this also asserts on the `selected` prop, this group could be renamed to `selection` or similar.